### DISCUSSION TITLE
"ImportError: No module named pycozo.ipyext_direct" and failed execution

### DISCUSSION THREAD

**Discussion by gisborne** [cozodb/cozo#171] [Q&A]
I get the error "ImportError: No module named pycozo.ipyext_direct" and none of the code blocks execute when I try to use Cozo from Jupyter.

This is within IntelliJ on a Apple Silicon machine.

Advice?

**Comment by zh217** [comment]
Does it work with Jupyter notebooks? I have not tested with IntelliJ. Apple Silicon isn't the problem here.

**Reply by gisborne** [reply]
It worked in a local Jupyter notebook. But the current tutorial on the site, which is generated from the notebook, shows failures instead of results.

**Comment by gisborne** [comment]
The problem appears in the tutorial you were gracious enough to let me update.
